source/manual/get-started.html.md

```diff
@@ -204,6 +210,13 @@ If installing the Python dependencies for fabric-scripts fails, your version of
 
 👉 Then, [set up your CLI access for AWS](/manual/aws-cli-access.html)
 
+NB, this is another candidate for adding to your `~/.bash_profile_vm`:
+
+```sh
+# Set up govukcli alias
+sudo ln -sf /var/govuk/govuk-aws/tools/govukcli /usr/local/bin/govukcli
+```
```

Comment:
How about just adding this to the PATH by default (through govuk-puppet)?

Reply:
I did try doing this but couldn't figure out where to put it - could you give any pointers please? :)

Reply:
Maybe do something similar to https://github.com/alphagov/govuk-puppet/commit/eb4135413c87844f85c3fa1abee68bf595991797

Reply:
Raised https://github.com/alphagov/govuk-puppet/pull/9209.